cmp/grid/Grid.js

```diff
@@ -453,9 +451,11 @@ class LocalModel {
             {externalSort} = this.model.experimental;
 
         return {
-            track: () => [agGridModel.agApi, this.model.sortBy],
+            track: () => [agGridModel.agColumnApi, this.model.sortBy],
             run: ([api, sortBy]) => {
```

Comment:
Should probably change `api` to `colApi` for consistency with other code

Reply:
Done.